app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -88,51 +88,32 @@ You can choose up to 4 teacher training courses for each application.
 
 If you do not find what you were looking for, there are [other ways to train](/train-to-be-a-teacher).
 
-Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-teacher-training-personal-statement).
+Before you start, it’s particularly important to think about your [teacher training personal statement](/train-to-be-a-teacher/how-to-apply-for-teacher-training#writing-your-personal-statement).
 
 Some people do not get on the course they want because their application has
 not convinced the admissions team that they’re right for it.
 
-## Writing your teacher training personal statement
+## Writing your personal statement
 
-Your teacher training personal statement explains why you want to be a teacher. It’s your chance to show your motivation, commitment and teaching potential. It’s a crucial part of your application - so it’s worth taking your time on it.
+Your personal statement explains why you want to be a teacher. It’s a crucial part of your application - so it’s worth taking your time on it.
```

Comment:
```suggestion
Your personal statement explains why you want to be a teacher. It’s a crucial part of your application, so it’s worth taking your time on it.
```
dash just felt a bit weird to me here (though I know it was already like this before)